documentation/modules/overview/con-key-features-kafka-connect.adoc

```diff
@@ -0,0 +1,141 @@
+. The sink connector creates the tasks to stream data
+. Tasks run in parallel to poll Kafka and return records
+. Converters put the records into a format suitable for the external data system
+. Transforms adjust the records, such as filtering or relabelling them
+. The sink connector is managed using KafkaConnectors or the Kafka Connect API
+
+== Tasks
+
+Data transfer orchestrated by the Kafka Connect runtime is split into tasks that run in parallel.
+A task is started using the configuration supplied by a connector instance.
+Kafka Connect distributes tasks to workers.
+
+* A source connector task polls the external data system and returns a list of records that a worker sends to the Kafka brokers.
+* A sink connector task receives Kafka records from a worker for writing to the external data system.
+
+If a task fails, it can be restarted through the Kafka Connect API.
+
+For sink connectors, the number of tasks created relates to the number of partitions being consumed.
+For source connectors, how the source data is partitioned is defined by the connector.
+You can control the maximum number of tasks that can run in parallel by setting `tasksMax` in the connector configuration.
+
+== Workers
+
+Workers store the initial connector configuration deployed to the Kafka Connect cluster.
+They also start connectors and their tasks.
+
+* Source connector workers read events from an external data system and use converters to construct key and value pairs to store in Kafka.
+* Sink connector workers read records from Kafka and use converters to construct records to pass into an external data system.
+
+A Kafka Connect cluster contains a group of workers with the same `group.id`.
+The ID identifies the cluster within Kafka.
+The ID is assigned in the worker configuration through the `KafkaConnect` resource.
+Worker configuration also specifies the names of internal Kafka Connect topics.
+The topics store connector offset and status information.
+The group ID and names of these topics must also be unique to the Kafka Connect cluster.
+
+Workers are assigned one or more connector instances and tasks.
+The distributed approach to deploying Kafka Connect is fault tolerant and scalable.
+If a worker pod fails, active workers share the tasks.
+You can add to a group of worker pods through configuration of the `replicas` property in the `KafkaConnect` resource.
+
+== Transforms
+
+Kafka Connect translates and transforms external data.
+Single-message transforms change messages into a format suitable for the target destination.
+For example, a transform might insert or rename a field. Transforms can also filter and route data.
+Plugins contain the implementation required for workers to perform one or more transformations.
+
+* Source connectors transform data before converting it into a format supported by Kafka.
+* Sink connectors transform data after converting it into a format suitable for an external data system.
```

Comment:
It might be clearer to explain when transforms are applied with respect to converters.